app/controllers/new_document_controller.rb

```diff
@@ -48,11 +49,11 @@ def options
   private
 
     def supertype_options
-      YAML.load_file("app/formats/supertypes.yml").map do |supertype|
+      SupertypeSchema.all.map do |supertype|
```

Comment:
Not related to this PR, but could we put this in the view for now, to be consistent `choose_document_type.html.erb`?

Reply:
Which parts would you put in the view? The entire class?

Reply:
It's the map part - for doc types we're doing a map in the view, whereas for supertypes we're doing a map in the controller. It seems better to do it in the view, as it's view-specific.

Reply:
Not sure how I would put this in a view though? We're doing the mapping inside a class:

https://github.com/alphagov/content-publisher/blob/903b6f29decdd3429dac01248d7d56c62339516b/app/controllers/new_document_controller.rb#L35-L61

Reply:
Oh, I meant just this bit (assigning @supertype_schemas in the controller).

```
      @supertype_schemas.all.map do |supertype|
        {
          value: supertype.id,
          text: supertype.label,
          hint_text: supertype.description,
          bold: true,
        }
      end
    end
```

Compare with line 7 of choose_document_types.html.erb.

Reply:
Ah, sorry, just understood. I guess it'll get refactored anyway as part of the SupertypeThing.